### PR TITLE
Update team.vue

### PR DESCRIPTION
**Pull request by MarcelCoding**
**Description**
Updated the Team Members on the Website

### CODE REVIEW

**Comment by MarcelCoding** [comment]
@JannikEmmerich Updated

File: pages/team.vue [hunk 1]
```diff
@@ -19,15 +19,14 @@
 
       <div class="group">
         <h6>Backend</h6>
-        <TeamMember github-name="TheCataliasTNT2k" name="TNT2k" role="Head of Backend"></TeamMember>
-        <TeamMember github-name="JannikEmmerich" name="Jannik" role="Head Assistant"></TeamMember>
-        <TeamMember github-name="MarcelCoding" name="Marcel" role="Contributor"></TeamMember>
+        <TeamMember github-name="TheCataliasTNT2k" name="TNT2k" role="Head of Python"></TeamMember>
```
Comment (JannikEmmerich) [inline pages/team.vue:22]:
```suggestion
        <TeamMember github-name="TheCataliasTNT2k" name="TNT2k" role="Head of Python Backend Development"></TeamMember>
```

File: pages/team.vue [hunk 2]
```diff
@@ -19,15 +19,14 @@
...
+        <TeamMember github-name="JannikEmmerich" name="Jannik" role="Head of Java"></TeamMember>
```
Comment (JannikEmmerich) [inline pages/team.vue:23]:
```suggestion
        <TeamMember github-name="JannikEmmerich" name="Jannik" role="Head of Java Backend Development"></TeamMember>
```

File: pages/team.vue [hunk 3]
```diff
@@ -19,15 +19,14 @@
...
+        <TeamMember github-name="MarcelCoding" name="Marcel" role="Head Assistant of Java"></TeamMember>
```
Comment (JannikEmmerich) [inline pages/team.vue:24]:
```suggestion
        <TeamMember github-name="MarcelCoding" name="Marcel" role="Head Assistant of Java Backend Development"></TeamMember>
```

**Pull request merged**